src/TabBarTop.js

```diff
@@ -27,30 +27,28 @@ const styles = StyleSheet.create({
   },
 });
 
-type DefaultProps = {
-  renderLabel: (scene: Scene) => ?React.Element<any>;
-}
-
 type Props = SceneRendererProps & {
   renderLabel: (scene: Scene) => React.Element<any>;
   indicatorStyle?: any;
+  labelStyle?: any;
 }
 
-export default class TabBarTop extends Component<DefaultProps, Props, void> {
+export default class TabBarTop extends Component<Props, void> {
```

Comment:
this should be `export default class TabBarTop extends Component<void, Props, void> {`